support/mock/mockrt.go

```diff
@@ -690,6 +697,11 @@ func (rt *Runtime) VerifyAggregateSeals(agg proof.AggregateSealVerifyProofAndInf
 	return nil
 }
 
+func (rt *Runtime) VerifyReplicaUpdate(replicaInfo proof.ReplicaUpdateInfo) error {
+	//TODO
```

Comment:
You can drop this looks fine for now.  If creating errors becomes important in tests you can come back to it.